torchok/losses/classification/binary_cross_entropy.py

```diff
@@ -0,0 +1,48 @@
+import json
+import logging
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from typing import Union
+
+from torchok.constructor import LOSSES
+
+
+@LOSSES.register_class
+class BCEWithLogitsLossX(nn.BCEWithLogitsLoss):
```

Comment:
Rename to BCEWithLogitsLoss as it extends the original BCE without changing other functionality